config.yml

```diff
@@ -0,0 +1,3 @@
+node_address: 0x1234567890123456789012345678901234567890
```

Comment:
1. Maybe put the following contents here so that the users can have a fully working example to run through if needed. Here is a proposed config.yml with some more detail: 
```yaml
# Note: Currently, this dashboard solution only allows the monitoring of one node at a time.

# eth mainnet example
node_address: 0x1BD61FAAa74F7B2aAF414753B74d6070f821D16D
chain_id: 1
rpc_endpoint: https://mainnet.gateway.tenderly.co/4X987cMYL0drsf7LIZXlSr

# eth holesky example 
# node_address: 0x40E1b90DD56f85C2207980d3751eF71B74CA7f9C
# chain_id: 17000
# rpc_endpoint: https://ethereum-holesky-rpc.publicnode.com
```

2. I know this is not scheduled for this release. But it may be nice for future iteration to allow users to add multiple nodes in sections like this:

```yaml
[node1]
node_address: ..
chain_id: ..
rpc_endpoint: ..

[node2]
node_address: ..
chain_id: ..
rpc_endpoint: ..
```